src/Detector/Hugo/HugoDetector.cs

```diff
@@ -42,22 +42,24 @@ public HugoDetector(ILogger<HugoDetector> logger)
         /// <inheritdoc/>
         public PlatformDetectorResult Detect(DetectorContext context)
         {
-            var isHugoApp = IsHugoApp(context.SourceRepo);
+            var isHugoApp = IsHugoApp(context.SourceRepo, out string directory);
             if (isHugoApp)
             {
                 return new PlatformDetectorResult
                 {
                     Platform = HugoConstants.PlatformName,
+                    Directory = directory,
```

Comment:
The name `Directory` does not really tell much. Can it be called `AppDirectory`?

Reply:
sure, will change it to `AppDirectory`